modules/ROOT/pages/introduction/resilience.adoc

```diff
@@ -0,0 +1,55 @@
+= Resilience
+:description: This section describes the resilience features of Ops Manager.
+
+Neo4j Ops Manager is equipped with multiple features to ensure resilient service.
+
+== Rate limiting
+To avoid the server being overloaded by one or a few clients, a rate limit is applied per IP address. This helps ensure that the server is always available to respond to requests by putting a limit on the load an individual connection is allowed to place on the system.
+
+The default configuration can be changed with the following server configuration parameters.
+
+[cols="<,<,<, <",options="header"]
+|===
+| Command line argument
+| Environment variable name
+| Description
+| Default value
+
+| `ratelimiter.period`
+| `RATELIMITER_PERIOD`
+| The amount time before the rate limiter resets the number of requests.
+| PT20S
+
+| `ratelimiter.limit_for_period`
+| `RATELIMITER_LIMIT_FOR_PERIOD`
+| Number of requests permitted (per IP) within the period.
+| 200
+
+| `ratelimiter.timeout_duration`
+| `RATELIMITER_TIMEOUT_DURATION`
+| When the limit is hit, wait this amount of time and check again.
+| PT10S
+|===
+
+== Circuit breaker
+
+Since query log capture can produce a vast amount of data depending on the workload and agent configuration, a so-called circuit breaker governs the reception of log data on the server side. If the amount of logs being received is vast enough to cause a degradation in handling of other messages, the circuit breaker will temporarily stop processing of query logs and assign full priority to other messages. Query log processing resumes automatically after some time has passed.
+
+The way to configure this behavior is to limit the amount of logs being transmitted by each agent. See *xref:../addition/agent-installation/self-registered.adoc#querylog[Query log collection configuration]*.
```

Comment:
Agent Query log configuration only specifies the interval to log the queries which might not give an estimate of how many queries that would result as some DBs might have higher frequency of queries right?

Reply:
Yeah I guess I could clarify that!